packages/@aws-cdk/aws-elasticloadbalancingv2/test/alb/target-group.test.ts

```diff
@@ -497,4 +497,18 @@ describe('tests', () => {
         app.synth();
       }).toThrow('Healthcheck interval 1 minute must be greater than the timeout 2 minutes');
     });
+
+  test('imported targetGroup has targetGroupName', () => {
+    // GIVEN
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app, 'Stack');
+
+    // WHEN
+    const importedTg = elbv2.ApplicationTargetGroup.fromTargetGroupAttributes(stack, 'importedTg', {
+      targetGroupArn: 'arn:aws:elasticloadbalancing:us-west-2:123456789012:targetgroup/myAlbTargetGroup/73e2d6bc24d8a067',
+    });
+
+    // THEN
+    expect(importedTg.targetGroupName).toEqual('myAlbTargetGroup');
```

Comment:
Is this really correct? Shouldn't the name be `'myAlbTargetGroup/73e2d6bc24d8a067'`?